src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -672,17 +678,23 @@ sealed internal class _SqlRPC
         internal int warningsIndexEnd;
         internal SqlErrorCollection warnings;
 
-        internal string GetCommandTextOrRpcName()
+        public SqlParameter GetParameterByIndex(int index, out byte options)
```

Comment:
Do we need to expose this API? can it be internal?

Reply:
Its public on an internal class which makes it effectively internal, but I've changed it to internal anyway.